workers/dockerfiles/Dockerfile.no_op

```diff
@@ -42,7 +42,9 @@ COPY workers/install_gene_convert.R .
 RUN Rscript install_gene_convert.R
 RUN mkdir -p gene_indexes
 WORKDIR /home/user/gene_indexes
-RUN curl -O https://zenodo.org/record/1327563/files/all_1533307773.zip
+ENV ID_REFINERY_URL https://zenodo.org/record/1322711/files/all_1532646318.zip
+RUN curl -O $ID_REFINERY_URL
+RUN echo $ID_REFINERY_URL > /etc/id_refinery_url.txt
```

Comment:
Chatted briefly with @dongbohu. Since we're going to make the parsing much simpler, the commands need to be something that is at least googleable. If this filename was `/etc/identifier_refinery_url` it may be a little bit friendlier. Then a user can google for "identifier refinery".